src/test/java/com/marklogic/kafka/connect/sink/SendConversionFailureRecordsToDlqKafkaTest.java

```diff
@@ -0,0 +1,134 @@
+package com.marklogic.kafka.connect.sink;
+
+import com.marklogic.client.io.SearchHandle;
+import com.marklogic.client.query.DeleteQueryDefinition;
+import com.marklogic.client.query.StructuredQueryBuilder;
+import com.marklogic.client.query.StructuredQueryDefinition;
+import com.marklogic.junit5.spring.SimpleTestConfig;
+import kafka.server.KafkaConfig$;
+import net.mguenther.kafka.junit.EmbeddedKafkaCluster;
+import net.mguenther.kafka.junit.KeyValue;
+import net.mguenther.kafka.junit.ReadKeyValues;
+import net.mguenther.kafka.junit.SendKeyValues;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+import static net.mguenther.kafka.junit.EmbeddedConnectConfig.kafkaConnect;
+import static net.mguenther.kafka.junit.EmbeddedKafkaCluster.provisionWith;
+import static net.mguenther.kafka.junit.EmbeddedKafkaClusterConfig.newClusterConfig;
+import static net.mguenther.kafka.junit.ObserveKeyValues.on;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class SendConversionFailureRecordsToDlqKafkaTest extends AbstractIntegrationSinkTest {
+
+    // Declared by AbstractSpringMarkLogicTest
+    @Autowired
+    private SimpleTestConfig testConfig;
+
+    private final String ML_COLLECTION = "kafka-data";
+    private final String TOPIC = "test-topic";
+    private final String DLQ_TOPIC = "test-dlq";
+    private final String KEY = String.format("key-%s", UUID.randomUUID());
+
+    private EmbeddedKafkaCluster kafka;
+
+    @BeforeEach
+    void setupKafka() {
+        provisionKafkaWithConnectAndMarkLogicConnector();
+        kafka.start();
+        cleanUpDataForTest();
+    }
+
+    @AfterEach
+    void tearDownKafka() {
+        kafka.stop();
+    }
+
+    @Test
+    void badMessagesShouldGoToTheDlq() throws InterruptedException {
+        Integer NUM_RECORDS = 2;
+        Integer NUM_BAD_RECORDS = 1;
+        sendSomeJsonMessages(NUM_RECORDS);
+        sendSomeBadJsonMessages(NUM_BAD_RECORDS);
```

Comment:
1) I changed the batchSize to 100 and the threadCount to 1, so that all the records would go at once.
2) However, the conversion error (at least the way I'm currently testing) is handled in Kafka code, not in my try-catch.

Reply:
Hmm - this test is seeming superfluous now, given that our connector only gets the 4 valid ones. I am thinking we should just drop this test - it would pass if we ran it before adding this story, so it's not verifying any new behavior. It's really only verifying that Kafka works. 

I think the `FailedConversionErrorReporterTest` is sufficient for us.